### PR TITLE
explain difference between client vs server navigation

### PR DESCRIPTION
**Pull request by Tomekmularczyk**
This PR references #28 . I have added a brief explanation about the difference between client vs server-side navigation.

Though, it may not be completely correct grammatically 🙈

### CODE REVIEW

**Comment by lfades** [comment]
@Tomekmularczyk I don't really get it, it may confuse people even more, can you instead explain what `<Link />` does at the end ? something like `<Link /> will prefetch the page so the navigation will only happen in the client without a server request and page refresh`

**Comment by Tomekmularczyk** [comment]
🤔  
@lfades how about:

> In order to support client-side navigation, we need to use Next.js's Link API, which is exported via `next/link`.
> The `<Link />` will prefetch the page and navigation will happen without server request and page refresh.

**Comment by lfades** [comment]
I like it, here's the same with some changes:
> <Link /> will prefetch the page and navigation will happen without a page refresh.

**Comment by Tomekmularczyk** [comment]
updated

File: learn/pages/learn/basics/navigate-between-pages/index.mdx [hunk 1]
```diff
@@ -23,6 +23,7 @@ Then we can access that page with http://localhost:3000/about.
 After that, we need to connect these pages. We could use an HTML "a" tag for that. However, it won't perform client-side navigation; it navigates to the page via the server, which is not what we want.
 
 In order to support client-side navigation, we need to use Next.js's Link API, which is exported via `next/link`.
+The `<Link />` will prefetch the page and navigation will happen without a page refresh.
```
Comment (lfades) [inline learn/pages/learn/basics/navigate-between-pages/index.mdx]:
```suggestion
`<Link />` will prefetch the page and navigation will happen without a page refresh.
```

**Comment by Tomekmularczyk** [comment]
image
Getting there...

**Pull request merged**